src/parsing/test/testing.ts

```diff
@@ -48,6 +48,17 @@ interface TestCase {
             appendDot?: boolean;
         };
     };
+    expectedErrors?: {
+        text: string,
+        token: string,
+        line: 0,
```

Comment:
Why do we need this line field if location has the first and last line in itself?